tasks/models.py

```diff
@@ -50,9 +50,14 @@ def filter_by_assignee(cls, assignee_id):
     def filter_by_symbol(cls, priority_filter):
         if not isinstance(priority_filter, Priority):
             raise ValueError
-        print(priority_filter)
         return cls.objects.filter(priority=priority_filter)
 
+    @classmethod
+    def filter_by_team(cls, team_filter):
+        if not isinstance(team_filter, Team):
+            raise ValueError("A valid team must be provided")
```

Comment:
I think that TypeError is a better fit here 
```suggestion
            raise TypeError("A valid team must be provided")
```